WordPicker.py

```diff
@@ -2,33 +2,31 @@
 import random
 
 class WordPicker(luigi.Task):
-
     def output(self):
-        return luigi.LocalTarget('output/word-picker/word.txt')
-    
+        return luigi.LocalTarget('./output/word-picker/word.txt')
```

Comment:
@JackDanger - I don't think this was actually the issue (just adding it to be explicit). There were some strange things going down with lines 13-14 (red).